utils/android-push.sh

```diff
@@ -18,7 +18,7 @@ if type adb > /dev/null 2>/dev/null && adb devices >/dev/null 2>/dev/null ; then
     adb push "../$XPI_NAME" /sdcard/"$XPI_NAME"
     adb shell am start -a android.intent.action.VIEW \
                        -c android.intent.category.DEFAULT \
-                       -d file:///mnt/sdcard/"$XPI_NAME" \
```

Comment:
We want Firefox to open a new tab to the folder rather than directly to the apk file.

This is because Firefox no longer prompts to install addons from direct links, it only does so if you manually click on the link to the APK from the directory listing